config/routes.rb

```diff
@@ -3,7 +3,12 @@
 Rails.application.routes.draw do
   namespace :api do
     namespace :v1 do
-      resources :products
+      resources :products, param: :id do
+        member do
+          put '/assign_product_details', to: 'products#assign_product_details'
```

Comment:
As Ticket TS-708 states in the technical specs:
The endpoint should be: **POST /api/v1/products/:product_id/assign_attributes.**
At the moment the endpoint is: **PUT /api/v1/products/:id/assign_product_details.**

The specs state further:
The endpoint: **GET /api/v1/products/:product_id** should show the **product_details**.
At the moment the endpoint is the following: **GET /api/v1/products/:id**

This probably would still work, as both is meaning the same. (id of product)